communication/vehicleconnections/mavsdkstation.cpp

```diff
@@ -80,3 +85,25 @@ QList<QSharedPointer<MavsdkVehicleConnection>> MavsdkStation::getVehicleConnecti
 {
     return mVehicleConnectionMap.values();
 }
+
+void MavsdkStation::on_timeout()
+{
+    for(int i = 0; i < vehicleTimers.size(); i++) {
```

Comment:
Would suggest to use a C++11 style loop for readability like:
```
for (const auto& vehicleTimer : vehicleTimers) {
 vehicleTimer.second++
...
```

Reply:
"const auto&" causes error, ok with just "auto&"?